src/components/TextArea/TextArea.jsx

```diff
@@ -52,15 +68,22 @@ TextArea.propTypes = {
   /** The id to link the textarea with the label */
   id: PropTypes.string.isRequired,
   /** If the textarea should take the height of the parent div */
-  fullHeight: PropTypes.bool
+  fullHeight: PropTypes.bool,
+  /** 
+   * this consumed by the default export that is wrapping the component into a ForwardRef
+   * @ignore
+   */
+  forwardRef: PropTypes.oneOfType([
```

Comment:
Hey @gmzjuliana I might be wrong but I think we can directly use `ref` as the prop name in here, if that is possible I think that is preferable as it's a bit more explicit 😉

Reply:
I gave it a quick try but I get an error message about special props 🤔 I'm gonna merge this one and maybe we can take a look later to see if there was something I might've missed :)

Reply:
I think we need to change the Class to extend from `React.forwardRef` to make it work https://reactjs.org/docs/forwarding-refs.html

Reply:
just a note for transparency -> Fede and I looked at this today and decided to leave it like this for now because the switch to ref seems to include a bit more work than it's worth right now